clustercontroller-core/src/main/java/com/yahoo/vespa/clustercontroller/core/hostinfo/ResourceUsage.java

```diff
@@ -11,26 +11,33 @@
  */
 public class ResourceUsage {
     private final Double usage;
+    private final String name;
```

Comment:
We could consider using Optional here, but I guess that doesn't work well with the @JsonProperty annotations used in the constructor.

Reply:
Will consider this for a later follow-up